web/src/pages/suppRequests/ApproveDoc.tsx

```diff
@@ -1,42 +1,69 @@
 // (c) Tecnologico de Monterrey 2022, rights reserved.
 
-import RequestCard from "../components/RequestCard/RequestCard";
+import RequestCard from "../../components/RequestCard/RequestCard";
 import back from "../../assets/background/login.png";
+import { useLazyLoadQuery } from "react-relay";
+import graphql from "babel-plugin-relay/macro";
+
+import { ApproveDocQuery, ApproveDocQuery$data } from "./__generated__/ApproveDocQuery.graphql";
 
 const ApproveDoc = (): JSX.Element => {
+  const data: ApproveDocQuery$data = useLazyLoadQuery<ApproveDocQuery>(
+    graphql`
+      query ApproveDocQuery($application_status_id: ID!) {
+        applicationByStatusID(application_status_id: $application_status_id) {
+          title
+          id
+          user {
+            id
+            name
+            first_lastname
+            second_lastname
+          }
+          citation {
+            id
+            title
+          }
+        }
+      }
+    `,
+    { application_status_id: "58a765a1-1204-48a5-b81c-c8c711aceb18" },
+  );
+
```

Comment:
This change will **break produciton**. Fixed IDs should **never** be implemented.